### ISSUE TITLE
Freezes on Chromium

### ISSUE THREAD

**Issue by jonholdsworth** [kylepaulsen/SuperBookmarkDesktop#12]
This looks like a great project.
But it loads, I can see the desktop, I get either the startup menu or nothing (right-click menu works) ... then... it freezes.
No activity at all.
I have not opened the JS console, please advise if you want more info or if you are already on this.

Maybe I'm pushing the envelope a bit. Chromium rather than Chrome and probably older (I hate upgrading, it always breaks something irreversibly).
Version 83.0.4103.61 (Official Build) Built on Ubuntu , running on Ubuntu 18.04 (64-bit)

**Comment by kylepaulsen** [comment]
I'm not aware of any problems. I use this extension every day and haven't seen a critical bug since last update.
I would definitely appreciate any console errors or any other details you might have. Open chrome dev tools (F12 or ctrl+shift+i) and go to the console tab for errors.

**Comment by jonholdsworth** [comment]
Sadly I can't even get the console to come up :( I'll keep trying and see if I can get you a report. Thanks for getting back.

**Comment by jonholdsworth** [comment]
Nope, it just doesn't seem to work on Chromium at all. I'm on the latest Linux Chromium now - 
Version 91.0.4472.101 (Official Build) Built on Ubuntu , running on Ubuntu 18.04 (64-bit)
I still can't get the console to show me anything, so for now it's just going in the too-hard basket.
If I ever get some insight I'll paste it here.

**Comment by kylepaulsen** [comment]
Installed a virtual machine of ubuntu and installed Chromium with similar versions (Ubuntu 18.04 64-bit, Chromium Version 91.0.4472.114 (Official Build))... unfortunately didnt see anything break... 😦 
![image](https://user-images.githubusercontent.com/865580/122866114-4bb93880-d2dc-11eb-9bc7-fd3f83676770.png)

I'm wondering if it has something to do with a particular bookmark you have, or maybe the quantity? How many bookmarks do you have? Maybe it's somehow too much for it to handle?

Edit: I tried importing 1000 bookmarks. It was def slow, but it didn't crash... At a loss at what's going on.

**Comment by jonholdsworth** [comment]
Ah! I have a lottttt of bookmarks, it would be easily in the thousands (or over A thousand). Chrome's default bookmark manager is pretty unwieldy and awful and yours was the only manager I saw that I actually liked - because I wanted to get sorting on that quantity of bookmarks!
So... yes, that's quite possibly it.

**Comment by jonholdsworth** [comment]
Forgot to add - I tested with all other extensions OFF and cache cleared, same result.

**Comment by jonholdsworth** [comment]
I'll try in Chrome, with and without all my bookmarks exported to it.

**Comment by kylepaulsen** [comment]
I'm guessing it's the quantity of bookmarks then... I'll try and think about how to handle 1000s of bookmarks.
